index.js

```diff
@@ -30,6 +30,22 @@ function attachComments(text, ast, opts) {
   return astComments;
 }
 
+// Cache compiled regexps in the likely case the same pragma is used for many
+// files.
```

Comment:
No seriously, this is not needed. Creating a dynamic regex isn't going to be a bottleneck.